ui/StatusQ/src/StatusQ/Components/StatusSmartIdenticon.qml

```diff
@@ -65,15 +69,20 @@ Loader {
 
     Component {
         id: roundedIcon
-        StatusRoundIcon {
-            asset.bgWidth: root.asset.bgWidth
-            asset.bgHeight: root.asset.bgHeight
-            asset.bgColor: root.asset.bgColor
-            asset.width: root.asset.width
-            asset.height: root.asset.height
-            asset.name: root.asset.name
-            asset.rotation: root.asset.rotation
-            asset.color: root.asset.color
+
+        StatusRoundButton {
+            width: root.asset.bgWidth
+            height: root.asset.bgHeight
+            color: root.asset.bgColor
+            icon.width: root.asset.width
+            icon.height: root.asset.height
+            icon.name: root.asset.name
+            icon.rotation: root.asset.rotation
+            icon.color: root.asset.color
+            icon.hoverColor: root.asset.hoverColor
```

Comment:
`icon` and `asset` are both `StatusAssetSettings`. We can reduce the bindings with `icon: root.asset`